jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/importer/parser/PayloadParser.java

```diff
@@ -90,4 +105,35 @@ public void writeTo(OutputStream os) throws IOException {
     os.write(payload, 0, payloadLength);
   }
 
+
+
+   public byte[] removeHeader(byte[] packet) {
+      String hexPacket = HexUtils.toHexString(packet);
+      String hexPacketParsed = "";
+      for (String key : msgStartFlags.keySet()) {
+         String startFlag = msgStartFlags.get(key);
+         int startIndex = hexPacket.toLowerCase().indexOf(startFlag);
+         if (hexPacketParsed == ""){
+            logger.debug("Start index for: " + key + " is: " + startIndex);
+            if (startIndex == -1) {
+               logger.debug("Message does not have header for: " + key);
+            } else if (startIndex <= HEADER_SIZE) {
+               logger.debug("Message has supported header. startIndex: " + startIndex + " msgFlag: " + startFlag);
+               hexPacketParsed = hexPacket;
+            } else if (startIndex > HEADER_SIZE && startIndex < 35) {
```

Comment:
The 35 is fairly arbitrary looking from a perspective of having no clue about packet headers. May be a good idea to have a comment here describing the logic here.